src/app/space-wizard/components/app-generator-single-selection-dropdown/app-generator-filtered-dropdown.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="combobox-container" >
+  <input type="hidden" name value>
+  <div [class.width-100]="field.display.enabled===false" class="input-group">
+    <input #inputFilter type="text" title="{{field.display.label}}" [class.form-control-danger]="!field.display.valid" id="{{field.name}}"
+      name="{{field.name}}" class="user-select-none combobox form-control" (keyup)="filterList(field, inputFilter.value)"
+      [disabled]="field.display.enabled===false" [(ngModel)]="field.display.text"
+      placeholder="select an option">
+    <ul class="dropdown-menu" [class.open]="field.display.open">
+      <li class="choice-container" (click)="field.display.text=choice.name;field.display.note=choice.description;field.value=choice.id;field.display.open=false;appGenerator.validate();"
+        *ngFor="let choice of field.display.choices|visibleItems">
+        <div [class.selected]="field.value === choice.id" class="dropdown-menu-item user-select-none cursor-pointer">
+          <div [class.vertical-layout]="choice.verticalLayout===true" class="choice-item">
+            <div *ngIf="choice.hasIcon===true" class="choice-icon margin-right-standard-offset">
+              <div [ngClass]="[choice.icon]"></div>
+            </div>
+            <div class="choice-content">
+              <div class="item-name">{{choice.name}}</div>
+              <div *ngIf="choice.description" class="item-description">{{choice.description||''}}</div>
+            </div>
+          </div>
+        </div>
+      </li>
+    </ul>
+    <span *ngIf="field.display.enabled" class="input-group-addon" (click)="field.display.open =!field.display.open">
+      <span class="caret"></span>
+      <span class="glyphicon glyphicon-remove"></span>
```

Comment:
please change to `<span class="pficon pficon-close"></span>` to use the PatternFly icon. Thanks!